src/css/styles.css

```diff
@@ -0,0 +1 @@
+
```

Comment:
you could add a style just to have an example and verify that it works, something like
```
body {
    background-color: brown;
}
```

Reply:
Thanks  for your feedback it was added 😀